table.csv

```diff
@@ -447,3 +447,8 @@ leofcoin-pr,          Leofcoin Peer Reputation,                         0x83
 torrent-info,         Torrent file info field (bencoded),               0x7b
 torrent-file,         Torrent file (bencoded),                          0x7c
 ed25519-pub,          Ed25519 public key,                               0xed
+
+Content Namespaces,,
+ipld-ns,              IPLD CID and optional path,                       0xee
```

Comment:
We can be a bit more abstract here and just say "IPLD path", "IPFS path", and "Swarm path" instead of specifying the precise format for the argument. That is, we know we'll use `<namespace-codec><stuff...>` but we can decide on `<stuff...>` later.

For example, swarm probably won't want arbitrary paths (it doesn't support directories, as far as I know. Instead, it will at *most* want byte ranges (e.g., `/swarm/CID` or `/swarm/CID/0..200`).

Reply:
Swarm does support paths, in a similar fashion to IPFS, by storing manifest files.

Reply:
Ah, got it. I just saw the docs on Swarm-Hash and assumed it was blob-only.

Reply:
In that case, I there is some value in trying to standardize on a binary format for content paths up-front. My thinking was just `<namespace-codec (varint)><binary-cid><string-path>` (where `<string-path>` is an optional `/a/b/c...`). Thoughts?

(we can also discuss this in a separate issue if you'd like).

Reply:
That seems reasonable. I liked the suggestion of reserving `/` as the multicodec for "path component". I think this'd be best discussed in its  own issue, though.